src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupParentID.class.php

```diff
@@ -45,19 +45,21 @@ protected function getLinkToParentRecordIfSet()
         $item->Load($this->data);
         $showLinkToParentRecord = $this->oDefinition->GetFieldtypeConfigKey('bShowLinkToParentRecord');
 
-        $html = $this->_GetHiddenField().$item->GetName();
+        $html = $this->_GetHiddenField();
+        $parentName = $item->GetName();
```

Comment:
What do you think of naming the variable `$itemName`? This would be in line with `$item`.

Also I'd move the declaration directly before its usage, so it's easier to relate declaration and usage (and there's also a really slight performance gain when we don't enter the if block).

Reply:
OK, done